modules/shortcodes/js/jetpack-email-subscribe.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
+ */
+/**
+
+/* global jQuery */
+/* jshint esversion: 6, es3:false */
+const JetpackEmailSubscribe = {
```

Comment:
It may be best to avoid the use of `const` here and everywhere else in the file. We try to stick to ES5 in other JS files that do not get compiled in Jetpack.

Reply:
YES! I would love that. I wrote that initially using `var`, but I got commit hook saying I should use `const`:

```
artpi~/GIT/jetpack(mailchimp/introduce-shortcode🔥 ) 😃 🚀  git add modules/shortcodes/js/jetpack-email-subscribe.js
artpi~/GIT/jetpack(mailchimp/introduce-shortcode) 😃 🚀  git commit -m "get rid of const"
Found '/Users/artpi/GIT/jetpack/.nvmrc' with version <10>
Now using node v10.13.0 (npm v6.4.1)
Found '/Users/artpi/GIT/jetpack/.nvmrc' with version <10>
Now using node v10.13.0 (npm v6.4.1)
husky > npm run -s precommit (node v10.13.0)


/Users/artpi/GIT/jetpack/modules/shortcodes/js/jetpack-email-subscribe.js
  11:1  error  Unexpected var, use let or const instead  no-var
  13:3  error  Unexpected var, use let or const instead  no-var
  17:3  error  Unexpected var, use let or const instead  no-var
  20:4  error  Unexpected var, use let or const instead  no-var
  22:4  error  Unexpected var, use let or const instead  no-var

✖ 5 problems (5 errors, 0 warnings)
COMMIT ABORTED: The linter reported some problems. If you are aware of them and it is OK, repeat the commit command with --no-verify to avoid this check.

husky > pre-commit hook failed (add --no-verify to bypass)

```

Now - is that my misconfiguration or the rules have changed?

Reply:
Done!